odata-to-abstract-sql.ometajs

```diff
@@ -850,19 +852,24 @@ var generateShortAliases = function(clientModel) {
 
 
 	var getRelationships = function(relationships) {
-		if (_.isArray(relationships)) {
-			return [];
-		} else {
-			return _(relationships).keys().reject(function(key) {
-				return key === '$';
-			}).union(
-				_.flatMap(relationships, getRelationships)
-			).value();
+		var relationshipKeys = Object.keys(relationships);
+		var nestedRelationships = [];
+        for (var i = 0; i < relationshipKeys.length; i++) {
```

Comment:
Wrong indentation here (seems to be spaces instead of tabs)

Reply:
That comes from me backporting a transpiled version :P  fixed